docs/security-self-assessment.md

```diff
@@ -0,0 +1,268 @@
+| SBOM | It's generated as part of every release, for example in [v.1.3.4](https://github.com/openfga/openfga/releases/download/v1.3.4/openfga_1.3.4_linux_386.tar.gz.sbom)|   |
+
+### Security Links
+
+  - https://github.com/openfga/openfga/security/policy
+  - https://github.com/orgs/openfga/security/risk
+
+## Overview
+
+OpenFGA is a high performance and flexible authorization/permission engine. It incorporates Relationship-Based Access Control (ReBAC) and Attribute Based Access Control (ABAC) concepts with a domain-specific language that makes it easy to craft authorization and permission solutions that can grow and evolve to any use case, at any scale. 
+
+It's inspired on the idea described in the [Google Zanzibar paper](https://research.google/pubs/pub48190/).
+
+### Background
+
+OpenFGA makes helps developers make authorization decisions by combining two concepts:
+
+- An Authorization Model, where developers define their authorization policies
+
+- A set of relationship tuples that instantiate the model and OpenFGA uses to answer access control queries.
+
+An authorization model looks like:
+
+```python
+model
+  schema 1.1
+
+type user
+type group
+  relations 
+    define member: [user]
+type folder
+  relations
+    define owner: [user]
+    define parent: [folder]
+    define viewer: [user, group#member] or owner or viewer from parent
+
+type document
+  relations 
+    define parent: [folder]
+    define owner: [user]
+    define viewer: [user, group#member] or owner or viewer from parent
+```
+
+Relationship tuples look like:
+
+| Subject | Relation | Object |
+| --- | --- | --- |
+| user:alice | member | group:engineering |
+| folder:root | parent | document:readme |
+| folder:root | viewer | group#engineering:member |
+| user:bob  | viewer | group#engineering:member |
+
+With this information, OpenFGA can be queried in different ways:
+
+When the  and 
+
+- Using the [/check](https://openfga.dev/api/service#/Relationship%20Queries/Check) endpoint to ask questions like "Is `user:alice` a `viewer` for `document:readme`?". With the data provided above, OpenFGA will return `{allowed : "true"}`, as Alice is a member of the engineering team, which has viewer access on the 'readme' document's parent folder.
+
+- Using the [/list-objects](https://openfga.dev/api/service#/Relationship%20Queries/ListObjects) endpoint to ask questions like "What are all the documents for which `user:alice` is a `viewer`. With the data provided above, OpenFGA will return `{object_ids { "document:readme" }`
+
+OpenFGA was accepted as a CNCF sandbox project in June 20212. Since our inclusion in the CNCF, it has been promising community contribution, growth, and adoption. Companies such as [Canonical](https://canonical.com) or [Okta](https://fga.dev) are building products and integrations on top of the OpenFGA, while others, like [Wolt](https://wolt.com) or [Read.ai](https://www.read.ai/) are using OpenFGA to implement authorization for their products.
```

Comment:
```suggestion
OpenFGA was accepted as a CNCF sandbox project in June 2022. Since our inclusion in the CNCF, it has been promising community contribution, growth, and adoption. Companies such as [Canonical](https://canonical.com) or [Okta](https://fga.dev) are building products and integrations on top of the OpenFGA, while others, like [Wolt](https://wolt.com) or [Read.ai](https://www.read.ai/) are using OpenFGA to implement authorization for their products.
```